File: app/src/main/java/com/intfocus/yonghuitest/kpi/entity/MDRPUnitCurveChartEntity.java
package com.intfocus.yonghuitest.kpi.entity;

import java.io.Serializable;
import java.util.ArrayList;

/**
 * 仪表盘实体对象
 * Created by zbaoliang on 17-4-28.
 */
public class MDRPUnitCurveChartEntity implements Serializable {
    public String chart_type;
    public String[] legend;
    public ArrayList<SeriesEntity> series;
    public String title;
    public String[] xAxis;
    public String[] yAxis;
    public int stateCode = 200;

    public class SeriesEntity implements Serializable {
        public String data;
        public String name;
        public String type;
    }
}

File: app/src/main/java/com/intfocus/yonghuitest/dashboard/kpi/entity/MDRPUnitCurveChartEntity.kt
package com.intfocus.yonghuitest.dashboard.kpi.entity

import java.io.Serializable
import java.util.ArrayList

/**
 * 仪表盘实体对象
 * Created by zbaoliang on 17-4-28.
 */
class MDRPUnitCurveChartEntity : Serializable {
    var chart_type: String? = null
    var legend: Array<String>? = null
    var series: ArrayList<SeriesEntity>? = null
    var title: String? = null
    var xAxis: Array<String>? = null
    var yAxis: Array<String>? = null
    var stateCode = 200

    inner class SeriesEntity : Serializable {
        var data: String? = null
        var name: String? = null
        var type: String? = null
    }
}